java/src/main/java/vowpalWabbit/learner/VWLearner.java

```diff
@@ -10,4 +10,6 @@
 
     // This tells the implementations that an IOException cannot be thrown.
     @Override void close();
+
+    void saveModel(String filename);
```

Comment:
I think the save interface should accept a `java.io.File` to show that it's a local File being saved.